office/kitlist/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem         1.0
 
 name               kitlist
-version            0.8.0
+version            1.1.0
 revision           1
```

Comment:
the `revision` should be reset to `0` when updating the version

Reply:
@reneeotten,  I have made the requested changes and corrected the commit message.

Many thanks,

Frank Dean